acquia/blt
# Fixes #3200: removes properly site alias on deployment.

Fixes #3200
--------

Changes proposed:
---------
(What are you proposing we change?)

- Clean up site alias after switching the context
-

Steps to replicate the issue:
----------
(If this PR is not fixing a defect/bug, you can remove this section.)

1. Create a standard multisite with blt
2. Setup site aliases, e.g. @site1.local, @site2.local
3. Deploy to cloud

Steps to verify the solution:
-----------
(How does someone verify that this does what you think does?)

1. When deployed, check the deployment logs on cloud interface. Deployment should not fail and the proper drush commands should *not* contain drush alias. E.g.:
```
/var/www/html/mysite.dev/vendor/bin/drush cache-rebuild --no-interaction -v --ansi in /var/www/html/mysite.dev/docroot
```
